electron/ipc/register/project.ts

```
@@ -41,6 +41,29 @@ function normalizeRecordingTimeOffsetMs(value: unknown): number {
+
+  const trimmedName = projectName.trim();
+  if (!trimmedName) {
+    return null;
+  }
+
+  const withoutExtension = trimmedName.replace(
+    new RegExp(`\\.${PROJECT_FILE_EXTENSION}$`, "i"),
+    "",
+  );
+  const sanitizedName = withoutExtension
+    .replace(/[<>:"/\\|?*\u0000-\u001F]/g, "")
```

Biome (2.4.11)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)